clients/go/pkg/auditopt/config.go

```diff
@@ -269,6 +246,15 @@ func labelsFromConfig(cfg *api.Config) audit.Option {
 	return audit.WithMutator(&lp)
 }
 
+func justificationFromConfig(ctx context.Context, cfg *api.Config) (audit.Option, error) {
+	jvsClient, err := client.NewJVSClient(ctx, &client.JVSConfig{JVSEndpoint: cfg.Justification.PublicKeysEndpoint})
```

Comment:
I found logging client uses `context.TODO()` https://github.com/abcxyz/lumberjack/blob/f03fda0c297d43a15ce2d367c44b7b1a7c233dfe/clients/go/pkg/auditopt/config.go#L250.
I'm wondering do we need to pass the context from the main function? Can we just use `context.TODO()` when we create the jvs client?

Reply:
I think passing the context is the right approach. When the context is canceled in main, the jvs client will be shutdown.